packages/cyberstorm/src/components/Layout/Developers/MarkdownPreview/MarkdownPreviewLayout.module.css

```diff
@@ -3,3 +3,47 @@
   flex-direction: column;
   gap: var(--gap--48);
 }
+
+.inputContainer {
+  border: 1px solid #333370;
+  border-radius: 0.5rem;
+}
+
+.inputContainer > textarea {
+  border-radius: 0.5rem 0.5rem 0 0;
+}
+
+.inputContainerFailure {
+  border-color: #f1385a;
+}
+
+.statusBar {
+  display: flex;
+  flex-direction: row;
+  gap: 0.75rem;
+  align-items: center;
+  padding: 0.75rem 1rem;
+  border-top: 1px solid var(--surface-7, #333370);
+  border-radius: 0 0 0.5rem 0.5rem;
+  color: #c6c3ff;
+  background: #0d0d21;
+}
+
+.icon {
+  width: 1em;
+  height: 1em;
+}
```

Comment:
The centering issue as discussed in relation to the icon component is present here. Just noting down as it will be most likely solved with another task.

Reply:
Is there an centering issue in this? I don't think there is?

Reply:
I'll note this down and go through the icons later 👍